e2e/cypress/integration/bot_accounts/crud_spec.js

```diff
@@ -0,0 +1,278 @@
+    it('MM-T1845 Create a new token via the UI', () => {
+        // # Visit the integrations
+        cy.visit(`/${newTeam.name}/integrations/bots`);
+
+        // * Check that the previously created bot is listed
+        cy.findByText(`Test Bot (@${botName})`).then((el) => {
+            // # Make sure it's on the screen
+            cy.wrap(el[0].parentElement.parentElement).scrollIntoView();
+
+            // # Click the 'Create token' button
+            cy.wrap(el[0].parentElement.parentElement).findByText('Create New Token').should('be.visible').click();
+
+            // # Try saving without description
+            cy.get('[data-testid=saveSetting]').click();
+            cy.wrap(el[0].parentElement.parentElement).find('input').scrollIntoView();
+
+            // * Check for error message
+            cy.get('#clientError').should('be.visible');
+
+            // # Add description
+            cy.wrap(el[0].parentElement.parentElement).find('input').click().type(botName + 'description!');
+
+            // # Save and check that no error is visible
+            cy.get('[data-testid=saveSetting]').click();
+
+            cy.get('#clientError').should('not.be.visible');
+
+            cy.findAllByText(botName + 'description!').should('exist');
+        });
+    });
+
+    it('MM-T1848 Delete Token', () => {
+        // # Visit the integrations
+        cy.visit(`/${newTeam.name}/integrations/bots`);
+
+        // * Check that the previously created bot is listed
+        cy.findByText(`Test Bot (@${botName})`).then((el) => {
+            // # Make sure it's on the screen
+            cy.wrap(el[0].parentElement.parentElement).scrollIntoView();
+
+            // # Click the 'Create token' button
+            cy.wrap(el[0].parentElement.parentElement).findByText('Create New Token').should('be.visible').click();
+
+            // # Add description
+            cy.wrap(el[0].parentElement.parentElement).find('input').click().type('description!');
+
+            // # Save
+            cy.get('[data-testid=saveSetting]').click();
+
+            // # Click Close button
+            cy.wrap(el[0].parentElement.parentElement).findByText('Close').should('be.visible').click();
+
+            cy.wrap(el[0].parentElement.parentElement).scrollIntoView();
+
+            // * Check that token is visible
+            cy.wrap(el[0].parentElement.parentElement).findByText(/Token ID:/).should('be.visible');
+
+            // # Click Delete button
+            cy.wrap(el[0].parentElement.parentElement).findByText('Delete').should('be.visible').click();
+
+            // * Validate that delete button is visible and click it
+            cy.get('#confirmModalButton').should('be.visible').click();
+
+            // * Check that token is not visible
+            cy.wrap(el[0].parentElement.parentElement).findByText(/Token ID:/).should('not.be.visible');
+        });
+    });
+
+    it('MM-T1849 Create a Personal Accees Token when email config is invalid', () => {
```

Comment:
```suggestion
    it('MM-T1849 Create a Personal Access Token when email config is invalid', () => {
```
Sorry about this one too.. it's now corrected in the test case.